_episodes_rmd/05-data-visualization.Rmd

```diff
@@ -196,8 +200,10 @@ mapping quality plot into three panels, one for each sample.
 
 ```{r first-facet, purl=FALSE}
 ggplot(data = variants, aes(x = POS, y = MQ, color = sample_id)) + 
- geom_point() + labs(x="Base Pair Position", y="Mapping Quality (MQ)") + 
-    facet_grid(~ sample_id)
+ geom_point() + 
+ labs(x = "Base Pair Position", 
+      y = "Mapping Quality (MQ)") + 
+ facet_grid(~ sample_id)
```

Comment:
I think you need the . here for `facet_grid()` as it is used in the later code below?

https://ggplot2.tidyverse.org/reference/facet_grid.html

Reply:
Thanks @naupaka. Updated!